apps/api/src/app/events/events.controller.ts

```diff
@@ -69,6 +71,8 @@ export class EventsController {
     @UserSession() user: IJwtPayload,
     @Body() body: TriggerEventRequestDto
   ): Promise<TriggerEventResponseDto> {
+    const mark = this.performanceService.buildEndpointTriggerEventMark(body.transactionId as string);
```

Comment:
maybe we should hold all this logic in the decorator? I can imagine it like this:
```
@MeasurePerformance('EventsController.trackEvent')
async trackEvent(
```
then I don't have to deal with the service

Reply:
@LetItRock  This is something nest-otel can cover for us